docs/introduction.rst

```diff
@@ -27,6 +27,6 @@ For a tutorial on using Sceptre, see Get Started, or find out more information a
 Code
 ----
 
-Sceptre's source code can be found on `BitBucket <https://bitbucket.org/cloudreach/sceptre/>`_. Note that the source code is currently closed source, and will only be visible to Cloudreach employees.
+Sceptre's source code can be found on `Github <https://github.com/cloudreach/sceptre/>`_. Note that the source code is currently closed source, and will only be visible to Cloudreach employees.
```

Comment:
Remove line 'Note that the source code is currently closed source, and will only be visible to Cloudreach employees.'